Signal/src/ViewControllers/ContactViewController.swift

```diff
@@ -427,7 +494,9 @@ class ContactViewController: OWSViewController, CNContactViewControllerDelegate
         nameLabel.autoPinTrailingToSuperviewMargin(withInset: hMargin)
 
         let valueLabel = UILabel()
-        valueLabel.text = value
+        if let value = value {
```

Comment:
no biggie, but can't `value` be a `String` not a `String?` ?

(since bestEffortFormatE164 returns a `String` (not a `String?`)

But `valueLabel.text=` accepts a String? anyway, so this unwrapping is doubly unnecessary. 🤓

Reply:
Removed.  And this method isn't just used to display phone numbers, so it does need to be optional.